app/controllers/Application.scala

```diff
@@ -59,8 +59,8 @@ class Application(
     Ok(views.html.unsupportedBrowserPage())
   }
 
-  def bundleLanding(title: String, id: String, js: String, newDesigns: String): Action[AnyContent] = CachedAction() { implicit request =>
-    if (newDesigns == "circles") {
+  def bundleLanding(title: String, id: String, js: String, INTCMP: String): Action[AnyContent] = CachedAction() { implicit request =>
```

Comment:
Something that I don't remember, is Play case sensitive for query params?

Reply:
Yes